MAUI/Shimmer/migration.md

```diff
@@ -0,0 +1,342 @@
+
+SfShimmer shimmer = new SfShimmer();
+this.Content = shimmer;
+shimmer.VerticalOptions = LayoutOptions.Fill;
+shimmer.SetBinding(SfShimmer.IsActiveProperty, "IsActive");
+
+var stackLayout = new StackLayout();
+
+var label = new Label();
+label.Text = "Content is loaded!";
+label.HorizontalOptions = LayoutOptions.CenterAndExpand;
+label.VerticalOptions = LayoutOptions.CenterAndExpand;
+
+stackLayout.Children.Add(label);
+shimmer.Content = stackLayout;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+</td>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<ContentPage
+xmlns:shimmer="clr-namespace:Syncfusion.Maui.Shimmer;assembly=Syncfusion.Maui.Core">
+
+<shimmer:SfShimmer x:Name="Shimmer" VerticalOptions="FillAndExpand">
+   <StackLayout>
+      <Label 
+         Text="Content is loaded!!"
+         HorizontalOptions="CenterAndExpand"
+         VerticalOptions="CenterAndExpand">
+      </Label>
+   </StackLayout>
+</shimmer:SfShimmer>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.Maui.Shimmer;
+...
+
+SfShimmer shimmer = new SfShimmer();
+this.Content = shimmer;
+shimmer.VerticalOptions = LayoutOptions.FillAndExpand;
+var stackLayout = new StackLayout();
+var label = new Label();
+label.Text = "Content is loaded!";
+label.HorizontalOptions = LayoutOptions.Fill;
+label.VerticalOptions = LayoutOptions.Fill;
+
+stackLayout.Children.Add(label);
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
+
+<table>
+<tr>
+<th>Xamarin SfShimmer</th>
+<th>.NET MAUI SfShimmer</th>
+<th>Description</th>
+</tr>
+<tr>
+<td>{{'[Content]'| markdownify }}</td>
+<td>Nil</td>
+<td>Not Supported.</td>
+</tr>
+<tr>
+<td>{{'[CustomView]'| markdownify }}</td>
+<td>{{'[CustomView]'| markdownify }}</td>
+<td>Gets or sets the custom view that is used for loading view in the Shimmer.</td>
+</tr>
+<tr>
+<td>{{'[WaveDirection]'| markdownify }}</td>
+<td>{{'[WaveDirection]'| markdownify }}</td>
+<td>Gets or sets the animation direction for Shimmer.</td>
+</tr>
+<tr>
+<td>{{'[Type]'| markdownify }}</td>
+<td>{{'[Type]'| markdownify }}</td>
+<td>Gets or sets the built-in shimmer view type.</td>
+</tr>
+<tr>
+<td>{{'[Color]'| markdownify }}</td>
+<td>{{'[Fill]'| markdownify }}</td>
+<td>Gets or sets the background color of shimmer view.</td>
+</tr>
+<tr>
+<td>{{'[WaveColor]'| markdownify }}</td>
+<td>{{'[WaveColor]'| markdownify }}</td>
+<td>Gets or sets the shimmer wave color.</td>
+</tr>
+<tr>
+<td>{{'[WaveWidth]'| markdownify }}</td>
+<td>{{'[WaveWidth]'| markdownify }}</td>
+<td>Gets or sets the width of the shimmer wave.</td>
+</tr>
+<tr>
+<td>{{'[AnimationDuration]'| markdownify }}</td>
+<td>{{'[AnimationDuration]'| markdownify }}</td>
+<td>Gets or sets the duration of the wave animation in milliseconds.</td>
+</tr>
+<tr>
+<td>{{'[IsActive]'| markdownify }}</td>
+<td>{{'[[IsActive]'| markdownify }}</td>
+<td>Gets or sets a value indicating whether to load actual content of shimmer.</td>
+</tr>
+<tr>
+<td>{{'[PersonaStyle]'| markdownify }}</td>
```

Comment:
Check this property as internal or public

Reply:
Removed the internal properties